packages/eslint-mdx/src/traverse.ts

```diff
@@ -117,7 +119,8 @@ export class Traverse {
             jsxNodes.length = 0
           }
         }
-      } else if (offset) {
+        // eslint-disable-next-line no-dupe-else-if
+      } else if (offset && is<Jsx>(node, 'jsx')) {
```

Comment:
I'm not sure how to handle this particular line, it looks like it could push content that is not `Jsx` into the `jsxNodes` list.
Is that correct? Is it intended?

Reply:
It's intended, for example:

```mdx
<div>

# Header

</div>

```